conformance/utils/http/http.go

```diff
@@ -117,6 +117,17 @@ func MakeRequestAndExpectEventuallyConsistentResponse(t *testing.T, r roundtripp
 	WaitForConsistentResponse(t, r, req, expected, timeoutConfig.RequiredConsecutiveSuccesses, timeoutConfig.MaxTimeToConsistency)
 }
 
+// MakeRequestAndExpectFailure makes a request with the given parameters.
+// This function needs to ensure that after the system is stable the Request is
+// not returning http 200 StatusCode.
+func MakeRequestAndExpectFailure(t *testing.T, r roundtripper.RoundTripper, timeoutConfig config.TimeoutConfig, gwAddr string, expected ExpectedResponse) {
+	t.Helper()
+
+	req := MakeRequest(t, &expected, gwAddr, "HTTP", "http")
+
+	WaitForConsistentFailureResponse(t, r, req, timeoutConfig.RequiredConsecutiveSuccesses, timeoutConfig.MaxTimeToConsistency)
```

Comment:
nit:
This doesn't wait for a consistent failure response, it only waits for one, so a better name would be WaitForFailureResponse.  I won't ask for a different timeoutConfig name (since this isn't about successes), but why not just use 1?
```suggestion
	WaitForFailureResponse(t, r, req, 1, timeoutConfig.MaxTimeToConsistency)
```

Reply:
Since this is a non-blocking comment and its for a test, let's change it to a const for now as you suggest and then consider this resolved. It's pretty easy to do a bit more refactoring of conformance tests after the fact.

Reply:
done, changed to `5`